patterns/2-structured/early-and-fluid-quality-management-for-predictable-delivery

```diff
@@ -0,0 +1,58 @@
+## Title
+
+Early and fluid quality management for predictable delivery
+
+## Patlet
+
+Testing practices are often short-cutted to meet deadlines leading to stability questions regarding the code. Frequent code review and testing is embedded in a smaller pull-request culture. 
+
+## Problem
+
+Code stability depends on how confident we are on our tests. In some cases, proper testing is not taking place until later stages (even pre-production deployment), generating delays in the delivery of the releases. Part of the problem is the size of changes. Bigger changes are more difficult to review and test. In other words, our development code is sent to pre-production without passing the tests, creating blocks and loops against pre-production environments while fixing the issues.
+
+## Forces
+
+Against:
+Time pressures by the business team.
+Perfectionism on the technical teams.
+Lack of confidence in the master branch of development, lack of stability.
+Needs investments to reap benefit cycle after cycle.
+
+In favor:
+Agility is a trend in both software development and project management (lean)
```

Comment:
I think that _perfectionism_ and _lack of confidence (instability)_ are forces in favor of this pattern.

```suggestion
Needs investments to reap benefit cycle after cycle.

In favor:
Agility is a trend in both software development and project management (lean)
Perfectionism on the technical teams.
Lack of confidence in the master branch of development, lack of stability.
```